Tests/TestUtils/TestBase.swift

```diff
@@ -65,18 +60,18 @@ class TestBase: XCTestCase {
         // to revisit when AMSDK-10169 is available
         // wait .2 seconds in case there are unexpected events that were in the dispatch process during cleanup
         usleep(200000)
-        resetTestExpectations()
+        resetTestExpectations(testNetworkService: nil)
         TestBase.isFirstRun = false
         EventHub.reset()
         UserDefaults.clearAll()
         FileManager.default.clearCache()
     }
     
     /// Reset event and network request expectations and drop the items received until this point
-    func resetTestExpectations() {
+    func resetTestExpectations(testNetworkService: TestNetworkService?) {
```

Comment:
If TestBase is not setting the network service then it should not reset it as well. With your changes, tests are setting the network service mock or server so they should reset it in their context.